CODEOWNERS

```diff
@@ -0,0 +1,36 @@
+# Core
+/ghcide @pepeiborra
+/hls-graph @pepeiborra
```

Comment:
miss `hie-compat` , `hls-plugin-api` and `hls-test-utils`

Reply:
`hie-compat` is further down. But yeah, I left off things I wasn't sure about. We don't *have* to have this for everything, I guess.

Reply:
well i would add at least hls-plugin-api and hls-test-utils, feel free to add me to both (i think @pepeiborra is a good candidate :-P)
and well given hls-plugin-api is the interface with lsp, maybe you could be a really goo candidate too :-P
Same for hls-test-utils (interface with lsp-test)

And what about docs?, maybe you and me could be in charge

Reply:
I don't actually know anything about `hls-plugin-api` or `hls-test-utils`... yet. So I'd be happy to put myself in maybe when I know them better. But docs, sure.

Reply:
iirc @berberman did quite nice improvements around config in hls-plugin-api

Reply:
Feel free to add me, though I was not being active as before, I still can take a look at these code. The initial code and idea are still in my mind :)

Reply:
Done.